tests/unit/test_tokens.py

```diff
@@ -9,6 +9,38 @@ def test_client_token(aau_token_path):
     assert exists(aau_token_path)
 
 
+def test_client_token_add_hostid(aau_token_path):
```

Comment:
tests the scenario where the client token has no host ID attached. It should replace the _file_ but leave the token _value_ intact.